crates/aranya-client-capi/output/aranya-client.h

```diff
@@ -1154,18 +1153,29 @@ AranyaError aranya_query_devices_on_team_ext(struct AranyaClient *client,
                                              struct AranyaExtError *__ext_err);
 
 /**
- * Writes the human-readable encoding of `device` to `str`.
+ * Writes the human-readable encoding of `id` to `str`.
+ *
+ * To always succeed, `str` must be at least `ARANYA_ID_STR_LEN` bytes long.
+ *
+ * @param device ID [`AranyaId`](@ref AranyaId).
+ * @param str ID string [`AranyaId`](@ref AranyaId).
+ * @param str_len returns the length of `str`
+ *
+ * @relates AranyaError.
```

Comment:
```suggestion
 * @relates AranyaId.
```

This makes it show up on the page for `AranyaId`, kinda like a method. Ditto for the existing one below.